apps/ledger-live-mobile/tsconfig.json

```diff
@@ -10,8 +10,11 @@
     "target": "esnext",
     "incremental": true,
     "baseUrl": ".",
+    "rootDir": ".",
     "paths": {
-      "@utils/*": ["./src/utils/*"]
+      "~/*": ["src/*"],
+      "@utils/*": ["./src/utils/*"],
+      "tests/*": ["src/__tests__/*"]
```

Comment:
Should we have a naming convention for this? :)

Reply:
I agree, I will go with `@` prefix